src/ocf_stats.c

```diff
@@ -55,6 +57,132 @@ static void ocf_stats_error_init(struct ocf_counters_error *stats)
 	env_atomic_set(&stats->write, 0);
 }
 
+static void _ocf_stats_block_update(struct ocf_counters_block *counters, int dir,
+		uint64_t bytes)
+{
+	switch (dir) {
+		case OCF_READ:
+			env_atomic64_add(bytes, &counters->read_bytes);
+			break;
+		case OCF_WRITE:
+			env_atomic64_add(bytes, &counters->write_bytes);
+			break;
+		default:
+			ENV_BUG();
+	}
+}
+
+void ocf_core_stats_vol_block_update(ocf_core_t core, ocf_part_id_t part_id,
+		int dir, uint64_t bytes)
+{
+	struct ocf_counters_block *counters =
+		&core->counters->part_counters[part_id].blocks;
+
+	_ocf_stats_block_update(counters, dir, bytes);
+}
+
+void ocf_core_stats_cache_block_update(ocf_core_t core, ocf_part_id_t part_id,
+		int dir, uint64_t bytes)
+{
+	struct ocf_counters_block *counters =
+		&core->counters->part_counters[part_id].cache_blocks;
+
+	_ocf_stats_block_update(counters, dir, bytes);
+}
+
+void ocf_core_stats_core_block_update(ocf_core_t core, ocf_part_id_t part_id,
+		int dir, uint64_t bytes)
+{
+	struct ocf_counters_block *counters =
+		&core->counters->part_counters[part_id].core_blocks;
+
+	_ocf_stats_block_update(counters, dir, bytes);
+}
+
+static void _ocf_core_stats_req_update(ocf_core_t core, ocf_part_id_t part_id,
```

Comment:
this function is used in exactly one place - how about moving the code to ocf_core_stats_request_update?
Similar comment applies to _ocf_core_stats_req_pt_update.